### PR TITLE
Update components.json

### PR DESCRIPTION
**Pull request by arthurgermano**
Added SCR - Svelte Client Router

### CODE REVIEW

File: src/pages/components/components.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
 [
+  {
+    "title": "svelte-client-router",
+    "url": "https://arthurgermano.github.io/svelte-client-router/#/svelte-client-router",
+    "npm": "https://www.npmjs.com/package/svelte-client-router",
```
Comment (MacFJA) [inline src/pages/components/components.json]: This should be only the NPM package name, not the NPM URL
```suggestion
    "npm": "svelte-client-router",
```

File: src/pages/components/components.json [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+    "description": "Svelte Client Router is everything you need and think when routing SPA's. This is The Svelte Router for SPA!",
+    "tags": [
+      "router",
+      "spa",
+      "routes",
+      "routing"
```
Comment (MacFJA) [inline src/pages/components/components.json]: The tags `router`, `spa`, `routes`, `routing` doesn't exists (yet?)

The existing tags that are close to yours are:
 - `routers`

----

For me the tag `spa` can be added, but for the others I'm not sure they add any "improvements".

File: src/pages/components/components.json [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+    ],
+    "addedOn": "2021-04-27T00:00:00Z",
+    "category": "Router and SPA"
```
Comment (MacFJA) [inline src/pages/components/components.json]: The category **`Router and SPA`** does not exists.
The closest are **`Routers`** or _Empty_ (<=> **Unclassified**)

**Review by MacFJA — CHANGES_REQUESTED** [review]

**Review by arthurgermano — COMMENTED** [review]
Fixed Issues

File: src/pages/components/components.json (hunk 2 above)
Comment (arthurgermano) [reply src/pages/components/components.json]: Updated to only routers and npm from link to name

File: src/pages/components/components.json (hunk 3 above)
Comment (arthurgermano) [reply src/pages/components/components.json]: Updated to routers categories only

File: src/pages/components/components.json [hunk 4]
```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "title: SCR Svelte Client Router",
```
Comment (MacFJA) [inline src/pages/components/components.json]: There are missing quotes (`"`) here
```suggestion
    "title": "SCR Svelte Client Router",
```

**Review by MacFJA — CHANGES_REQUESTED** [review]

**Review by arthurgermano — COMMENTED** [review]
Fixed problem with title

**Review by MacFJA — APPROVED** [review]

**Review by kevmodrome — APPROVED** [review]

**Pull request merged**